.github/workflows/e2e.yml

```diff
@@ -22,6 +22,7 @@ jobs:
     runs-on: ubuntu-18.04
     env:
       TTN_LW_LOG_LEVEL: debug
+      COCKROACHDB_COCKROACH_TAG: v19.2.12
```

Comment:
Please add TODO and refer to https://github.com/TheThingsNetwork/lorawan-stack/issues/4184 

Also, why is this version number different from the one you mention in the original comment?